getting-started-instructions.md

```diff
@@ -0,0 +1,50 @@
+# Setting up the GP2GP Adaptors within INT
+
+The below instructions assume you will be using a single message broker for all messages.
+
+### Setting up the MHS adaptor
+
+This is a __required__ prerequisite for both requesting and sending adaptors. 
+
+1. Identify a HSCN IP address(es) which can be used for the GP2GP process, and made available to the machine running the MHS Inbound.
+   This IP will be used by spine when initiating HTTPS requests into your environment.
```

Comment:
Can Spine be uppercased.
Also, all the bullet points are shown as number 1.

Reply:
Fixed the Spine uppercasing.

When rendering the numbers, Markdown ignores the numbers provided and creates its own values. The benefit of having the bullet points as 1 in the markdown means that when you avoid the fragility of renumbering the existing lines, want to add a new step.